vignettes/articles/pre-processing.Rmd

```diff
@@ -0,0 +1,83 @@
+---
+title: "Pre-Processing SomaScan"
+author: "Amanda Hiser and Caleb Scheidel, SomaLogic Operating Co., Inc."
+description: >
+  A primer on pre-processing 'SomaScan' data for analysis, including filtering
+  samples and features, data QC, and transformations.
+output:
+  rmarkdown::html_vignette:
+    fig_caption: yes
+vignette: >
+  %\VignetteIndexEntry{Pre-Processing SomaScan}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  chunk_output_type: console
+---
+
+```{r setup, include = FALSE}
+library(SomaDataIO)
+library(ggplot2)
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "figures/pre-processing-"
+)
+```
+
+
+# Overview
+
+`SomaDataIO` contains functionality to pre-process an `.adat` file to prepare
+RFU data for analysis.  The typical data analysis path includes several steps
+of pre-processing, including:
+
+1. Filtering features
+2. Filtering samples
+3. Data QC and Bias Checks
```

Comment:
I think the name of this section (as described here) should match the section header used below, just for consistency and to avoid any confusion